package.json

```diff
@@ -14,7 +14,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "pretest": "jshint .",
+    "pretest": "eslint .",
```

Comment:
Cross-posting from https://github.com/strongloop/generator-loopback/pull/174#discussion_r59085659: please move linter to "posttest" and move the line after "test".